packages/react-components/src/templates/UserProfileHeader.tsx

```diff
@@ -123,6 +123,14 @@ const lastModifiedNoContactStyles = css({
 const tabNavStyles = css({
   gridArea: 'tab-nav',
 });
+const avatarContainer = css({ display: 'grid', width: 90, height: 90 });
+const imageContainer = css({ gridRow: 1, gridColumn: '1 / span 2' });
```

Comment:
Does this actually need more than 1 row/column at all? it seems overlaying both in row1 col1 and then aligning and justifying should do the trick?